Plugins/Json/include/Acts/Plugins/Json/MapJsonConverter.hpp

```diff
@@ -0,0 +1,142 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/TrackingGeometry.hpp"
+#include "Acts/Material/ISurfaceMaterial.hpp"
+#include "Acts/Material/IVolumeMaterial.hpp"
+#include "Acts/Plugins/Json/GeometryHierarchyMapJsonConverter.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include <Acts/Geometry/TrackingVolume.hpp>
+#include <Acts/Surfaces/Surface.hpp>
+
+#include <map>
+
+#include <nlohmann/json.hpp>
+
+namespace Acts {
+
+using SurfaceAndMaterial =
+    std::pair<std::shared_ptr<const Acts::Surface>,
+              std::shared_ptr<const Acts::ISurfaceMaterial>>;
+using TrackingVolumeAndMaterial =
+    std::pair<const Acts::TrackingVolume*,
+              std::shared_ptr<const Acts::IVolumeMaterial>>;
+
+/// @class MapJsonConverter
+///
+/// @brief read the material from Json
```

Comment:
This only converts MaterialMaps, why not 'MaterialMapJsonConverter' ?

Reply:
Yeah you are right I will change that